checkov/terraform/checks/resource/aws/ALBListenerHTTPS.py

```diff
@@ -18,8 +18,10 @@ def scan_resource_conf(self, conf):
         :param conf: aws_kms_key configuration
         :return: <CheckResult>
         """
+        self.evaluated_keys = []
```

Comment:
This is done on all checks and seems redundant since it's done on the base. Is it necessary?

Reply:
Explained this on my third answer in previous comment, LMK what you think

Reply:
@arielkru I agree with @robeden . Let's put it in `BaseCheck`'s `run` method. Check out line 39

Reply:
I removed these from the checks and added it to base_check, @robeden thanks for the great input :)
 @nimrodkor PTAL